app/src/main/res/layout/alert_dialog_input.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+
+    <EditText
+        android:id="@+id/alert_dialog_edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:inputType="text"
+        android:maxLines="1"
+        android:backgroundTint="@color/dark_grey"
+        android:textCursorDrawable="@drawable/custom_cursor"
+        android:hint="..." />
```

Comment:
Maybe we can replace this with something more descriptive, like `My todo list` or `Your new description`?

Also, it should be a `@string` resource as the linter points out.